tests/mysql.rs

```diff
@@ -87,40 +87,6 @@ async fn pool_immediately_fails_with_db_error() -> anyhow::Result<()> {
     Ok(())
 }
 
-#[cfg(feature = "macros")]
```

Comment:
Oh doi, this was fine before.

The separate file is better anyway, thanks for taking the initiative.